kuka_kr10_support/launch/test_kr10r900_2.launch

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find kuka_kr10_support)/launch/load_kr10r900_2.launch" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
+    <param name="use_gui" value="true" />
+  </node>
```

Comment:
do we still want to support Kinetic? If not: we should probably move to using JSP GUI here.

Or we could migrate everything at once in a single PR.

I'm ok with either.

Reply:
currently there is only `indigo-devel` but I would be fine with opening a `melodic-devel` (opw etc)